python/test/unit/fem/test_assembler.py

```diff
@@ -1139,6 +1139,17 @@ def block():
 
         A0.destroy(), A1.destroy(), A2.destroy()
 
+    def test_block_null_lifting(self):
+        from dolfinx.fem.petsc import assemble_vector_block
+
+        comm = MPI.COMM_WORLD
+        msh = create_unit_square(comm, 2, 2)
+        V = functionspace(msh, ("Lagrange", 1))
+        W = functionspace(msh, ("Lagrange", 2))
+        v, w = ufl.TestFunction(V), ufl.TestFunction(W)
+        L = form([v * ufl.dx, w * ufl.dx])
```

Comment:
```suggestion
        L = form([ufl.conj(v) * ufl.dx, ufl.conj(w) * ufl.dx])
```
To support complex assembly